File: src/main/java/my/study/repository/design/pattern/bridge/java/coffee/LargeCoffee.java
package my.study.repository.design.pattern.bridge.java.coffee;

import my.study.repository.design.pattern.bridge.java.additive.BaseAdditive;

/**
 * @author djh on  2018/9/25 14:14
 * @E-Mail 1544579459@qq.com
 */
public class LargeCoffee extends BaseCoffee {

    public LargeCoffee(BaseAdditive baseAdditive) {
        super(baseAdditive);
    }

    @Override
    public void showCoffeeInfo() {
        System.out.println("大杯加 " + mBaseAdditive.toString() + " 咖啡");
    }
}

File: src/main/java/my/study/repository/design/pattern/bridge/kotlin/coffee/LargeCoffee.kt
package my.study.repository.design.pattern.bridge.kotlin.coffee

import my.study.repository.design.pattern.bridge.kotlin.additive.BaseAdditive


/**
 * @author djh on  2018/10/4 17:41
 * @E-Mail 1544579459@qq.com
 */
class LargeCoffee(baseAdditive: BaseAdditive) : BaseCoffee(baseAdditive) {

    override fun showCoffeeInfo() {
        println("大杯 $baseAdditive 咖啡")
    }
}